lib/brain.js

```diff
@@ -1485,6 +1497,13 @@ Brain.prototype._start = function _start () {
     minimumTx: this.billValidator.lowestBill(coin.minimumTx),
     cryptoNetwork: coin.cryptoNetwork
   }
+
+  if (!this.tx.termsAccepted && !this.trader.terms.active) {
+    // terms are automatically accepted
+    // if terms screen is not enabled
+    updateRec.termsAccepted = true
+  }
+
```

Comment:
We should not record that user accepted terms if he didn't, even if there are no terms. We should handle this case by checking whether terms are enabled to determine whether to show the screen. I would remove this section.

Reply:
noted